src/main/java/net/neoforged/neoforge/server/ServerLifecycleHooks.java

```diff
@@ -143,8 +159,13 @@ private static void runModifiers(final MinecraftServer server) {
                 .toList();
 
         // Apply sorted biome modifiers to each biome.
-        registries.registryOrThrow(Registries.BIOME).holders().forEach(biomeHolder -> {
-            biomeHolder.value().modifiableBiomeInfo().applyBiomeModifiers(biomeHolder, biomeModifiers);
+        final var biomeRegistry = registries.registryOrThrow(Registries.BIOME);
+        biomeRegistry.holders().forEach(biomeHolder -> {
+            stripKnownPack(
+                    biomeHolder.value().modifiableBiomeInfo()
+                            .applyBiomeModifiers(biomeHolder, biomeModifiers, registries),
+                    biomeHolder,
+                    biomeRegistry);
```

Comment:
This code can become quickly unreadable, I would highly prefer it if the `applyBiomeModifier` method call would be in an if-switch and the body of said switch call `stripKnownPack`